gdnative-core/src/export/property/hint.rs

```diff
@@ -130,13 +140,22 @@ impl EnumHint {
         let mut s = String::new();
 
         let mut iter = self.values.iter();
+        let write_item = |s: &mut String, item: &(String, Option<i64>)| match item {
+            (key, Some(val)) => {
+                write!(s, "{key}:{val}")
+            }
+            (key, None) => {
+                write!(s, "{key}")
+            }
+        };
```

Comment:
If the whole signature is going to be spelled out anyway it should really be a `fn` instead of a closure.